Include link to internal templates code (#1794)
gohugoio/hugoDocs: content/en/templates/internal.md

## The Internal Templates

> The code for these templates and more can be found here at the [internal templates](https://github.com/gohugoio/hugo/tree/master/tpl/tplimpl/embedded/templates):

Reviewer: ```suggestion
The code for these templates is located [here](https://github.com/gohugoio/hugo/tree/master/tpl/tplimpl/embedded/templates).
```
